python/cudf/cudf/core/index.py

```diff
@@ -859,6 +872,81 @@ def min(self):
     def max(self):
         return self._minmax("max")
 
+    @property
+    def values(self):
+        return cupy.arange(self.start, self.stop, self.step)
+
+    def to_frame(self, index=True, name=None):
+        """Create a DataFrame with a column containing this Index"""
+
+        if name is not None:
+            col_name = name
+        elif self.name is None:
+            col_name = 0
+        else:
+            col_name = self.name
+        return cudf.DataFrame(
+            {col_name: self._range}, index=self if index else None
+        )
+
+    def to_series(self, index=None, name=None):
+        """
+        Create a Series with both index and values equal to the index keys.
+        Useful with map for returning an indexer based on an index.
+
+        Parameters
+        ----------
+        index : Index, optional
+            Index of resulting Series. If None, defaults to original index.
+        name : str, optional
+            Name of resulting Series. If None, defaults to name of original
+            index.
+
+        Returns
+        -------
+        Series
+            The dtype will be based on the type of the Index values.
+        """
+        return cudf.Series._from_data(
+            self._data,
+            index=self.copy(deep=False) if index is None else index,
+            name=self.name if name is None else name,
+        )
+
+    def any(self):
+        """
+        Return whether any elements is True in Index.
+        """
+        return self._values.any()
```

Comment:
You should be able to implement this method without using `_values`. Under what circumstances can `any` return `False` for a `RangeIndex?